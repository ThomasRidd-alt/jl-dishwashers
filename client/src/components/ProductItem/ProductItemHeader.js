const ProductItemHeader = props => {
    return (
        <header className={'item-page__header'}>
            <h1 className={'item-page__heading'}>{props.title}</h1>
        </header>
    )
};

export default ProductItemHeader