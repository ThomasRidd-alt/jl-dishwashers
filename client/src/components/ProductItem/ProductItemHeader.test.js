import React from "react";
import {mount} from 'enzyme';
import ProductItemHeader from "./ProductItemHeader";

describe('ProductItemHeader', () => {
    const id = 1;
    const props = {
        title: 'Hello world'
    }
    const component = mount(<ProductItemHeader {...props}/>);

    it('renders properly', () => {
        expect(component).toMatchSnapshot()
    })

    it('renders as H1', () => {
        expect(component.find('h1').text()).toContain(props.title)
    })
})