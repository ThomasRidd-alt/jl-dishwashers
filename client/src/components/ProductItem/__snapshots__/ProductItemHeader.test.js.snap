// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductItemHeader renders properly 1`] = `ReactWrapper {}`;
